c7n/resources/rds.py

```diff
@@ -1426,6 +1426,7 @@ class RDSSnapshotDelete(BaseAction):
     permissions = ('rds:DeleteDBSnapshot',)
 
     def process(self, snapshots):
+        snapshots = self.filter_resources(snapshots, 'SnapshotType', ('manual',))
```

Comment:
we might also want a sanity check Ala `if not snapshots: return`

Reply:
Makes sense, done :+1: 

(Though returning `[]` rather than the implicit `None` to keep a consistent list return value... I remember the `None` return value biting us somewhere else though I forget the details...)